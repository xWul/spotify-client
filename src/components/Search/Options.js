/* LIBS */
import React from 'react'
import styled from 'styled-components'
/* CONFIG */
import { screens } from '../../theme/globalStyle'
/* COMPONENTS */
import Select from '../ui/Select'

const Content = styled.div`
  width: 40%;
  @media(max-width: ${screens.small}) {
    width: 100%
  }
`

const Options = (props) => {
  return (
    <Content>
      <Select
        {...props}
        value={props.selected}>
        <option value={'artist'}>Artists</option>
        <option value={'album'}>Albums</option>
        <option value={'track'}>Tracks</option>
      </Select>
    </Content>
  )
}

Options.defaultProps = {
  selected: 'artist'
}

export default Options
