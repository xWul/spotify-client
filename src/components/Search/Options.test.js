import React from 'react'
import renderer from 'react-test-renderer'
import Options from './Options'

test('Select renderer without crash with selected album', () => {
  const component = renderer.create(
    <Options selected={'album'} />
  )

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
