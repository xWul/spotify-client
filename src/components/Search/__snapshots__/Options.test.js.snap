// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renderer without crash with selected album 1`] = `
<div
  className="sc-htpNat farngo"
>
  <div
    className="sc-bdVaJa LDKxg"
  >
    <select
      className="sc-bwzfXH fGrHko"
      selected="album"
      value="album"
    >
      <option
        value="artist"
      >
        Artists
      </option>
      <option
        value="album"
      >
        Albums
      </option>
      <option
        value="track"
      >
        Tracks
      </option>
    </select>
  </div>
</div>
`;
